### PR TITLE
Change "Delete" to "Archive" in PDP

### LINKED ISSUE #1502: [PDP]  Use "Archive" label instead of "Deleted"
Replace "Deleted" with "Archive" in the product "delete" functionality.  (as the product isn't really deleted, Archive is more correct.)

### PR DESCRIPTION
**Pull request by zenweasel**
Resolves #1502

### CODE REVIEW

File: imports/plugins/included/product-variant/client/templates/products/productGrid/controls.html [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
       {{#if product.isDeleted}}
         <div>
-        <span class="badge badge-danger" data-i18n="app.deleted">
-          <span>Deleted</span>
+        <span class="badge badge-danger" data-i18n="app.archived">
+          <span>Archived</span>
```
Comment (aaronjudd) [inline imports/plugins/included/product-variant/client/templates/products/productGrid/controls.html]: Does this work? shouldn't `data-i18n="app.archived"` be on the inner span?

File: private/data/i18n/en.json [hunk 2]
```diff
@@ -355,6 +359,8 @@
         "restoreProduct": "Restore Product from Trash",
         "deleteThisProduct": "Delete this product?",
```
Comment (aaronjudd) [inline private/data/i18n/en.json]: We should consider removing this phrase, as it's not used anywhere now.

File: imports/plugins/included/product-variant/client/templates/products/productGrid/controls.html (hunk 1 above)
Comment (mikemurray) [inline imports/plugins/included/product-variant/client/templates/products/productGrid/controls.html]: This works. The inner span is because Blaze and jQuery fight each other for who has the right to update the text contents of an element. The `<span>` around the `Archived` prevents this from happening. Don't know why this is... but this solution seems to fix that issue.

**Comment by mikemurray** [comment]
Still tagged as deleted. They are archived as well.

screen shot 2016-10-20 at 2 13 38 pm

**Comment by zenweasel** [comment]
I didn't change these on purpose because I thought we were still deleting variants. If I accidentally add a variant I can't immediately delete it?

**Comment by mikemurray** [comment]
tl;dr No. 

Since variants are products, its all covered by the new revision control system, including all product hooks.

Meaning, it would just be published it as "deleted". There is a collection hook that blocks all deletion attempts for all product types unless you bypass the hook with `_direct`. That hooks assigns `isDeleted: true` instead.

**Comment by mikemurray** [comment]
The variant might no be visible on the product after you delete/archive it. but it's still around in database.

This is a case where we need some sort of list view for all products in the dashboard so we can see what's active / archived / what-ever-other-status

**Comment by zenweasel** [comment]
Ok. Out of scope for this PR obviously but maybe we could so a check that if something has no orders and has never been added to cart it could actually be deleted. I think we had talked about this at some point.

**Comment by zenweasel** [comment]
Should be reflecting Archived for variants as well

example-product

**Comment by mikemurray** [comment]
👍 

looks good to me!

**Comment by aaronjudd** [comment]
👍  If you archive / delete a variant, where on the product grid was previously showing a range, and you leave only one item, the grid is still showing price range based on variants (that are now archived). This might be related to #1519, but I'll create a new issue.  Otherwise from just a review of "does deleted now say archived", yes, this is accepted.
